ssas/hash_test.go

```diff
@@ -65,6 +67,35 @@ func (s *HashTestSuite) TestHashWithIter() {
 	s.True(h1.IsHashOf(val))
 }
 
+func TestHashIterTime(t *testing.T) {
+	// This test is not intended to run with the remain suite.  Is is solely used to determine the number of hash
+	// iterations needed to reach a 1 second duration. To invoke the test use:
+	// DEBUG=true  go test -v github.com/CMSgov/bcda-ssas-app/ssas -run TestHashIterTime
+
+	// Check for local dev environment
+	if os.Getenv("DEBUG") != "true" {
+		t.SkipNow()
```

Comment:
:+1: Very elegant way to selectively run our tests. By using skipNow(), we won't be running this test in our unit tests, but it's easy for users to quickly enable for local testing.